src/panels/config/integrations/integration-panels/zwave_js/zwave_js-config-dashboard.ts

```diff
@@ -419,11 +419,6 @@ class ZWaveJSConfigDashboard extends SubscribeMixin(LitElement) {
                       "ui.panel.config.zwave_js.common.rebuild_network_routes"
                     )}
                   </ha-button>
-                  <ha-button @click=${this._openOptionFlow}>
-                    ${this.hass.localize(
-                      "ui.panel.config.zwave_js.common.reconfigure_server"
```

Comment:
Do we no longer have an option to reconfigure the server?

Reply:
Is the translation key no longer used then?

Reply:
The option has moved to a menu in the flow like ZHA. The string comes from core now so we can indeed remove it.